tests/annotation_model_tests/test_annotation_model_nieuwkoopse_plassen.py

```diff
@@ -0,0 +1,192 @@
+import pickle
+import glob
+import rasterio
+import numpy as np
+import pandas as pd
+import contextlib
+from tif_model_iterator import tif_kernel_iterator
+from filenames.file_name_generator import OutputFileNameGenerator
+import io
+import sys
+
+# Adjust this for the tests!!
+model_path = "C:/repos/satellite-images-nso-datascience/saved_models/Superview_Nieuwkoopse_plassen_20190302_113613_to_20221012_104900_random_forest_classifier.sav"
+location = "Nieuwkoopse_plassen"
+satellite_constellation = "Superview"
+output_path_test = "E:/output/test/Nieuwkoopse_plassen/"
+output_path = "E:/output/test/Nieuwkoopse_plassen/"
+
+# Put in settings file
+test_tif_files_dir = "E:/output/test/Nieuwkoopse_plassen/*SV*.tif"
```

Comment:
There is a tests/annotation_model_tests/setttings.py, but I assume it is not being used then?